examples/frompapers/Izhikevich_2003.py

```diff
@@ -0,0 +1,179 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[1]:
+
+
+##Author: Akif Erdem Sağtekin
+
+##Implemented from "https://www.izhikevich.org/publications/spikes.htm"
+
+############################
+
+# I am in favor of coding simplicity for scientific computing when applicable.
+# Since there are few neural groups and connections in this simulation, 
+# all the neuron groups and synaptic connections are explicitly coded line by line, 
+# without even using any for loops. 
+
+############################
+
+from brian2 import *
+from brian2tools import *
+import numpy as np
+np.random.seed(1337)
```

Comment:
explain why the seed is necessary

Reply:
Just used during fine-tuning for weights, removed.